apps/web/src/pages/auth/components/QuestionnaireForm.tsx

```diff
@@ -75,6 +65,18 @@ export function QuestionnaireForm() {
     await login(organizationResponseToken);
     await refetchEnvironments();
 
+    try {
+      await api.post(`/v1/bridge/sync`, {
+        bridgeUrl: BRIDGE_SYNC_SAMPLE_ENDPOINT,
+        source: 'sample-workspace',
+      });
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e);
```

Comment:
Do we need this log?

Reply:
Yes, I think it makes sense to log it to console, otherwise will be silent